internal/ent/hooks/event.go

```diff
@@ -220,6 +228,38 @@ func RegisterListeners(e *Eventer) error {
 		return err
 	}
 
+	_, err = e.Emitter.On(OrganizationDelete, handleOrganizationDelete)
+	if err != nil {
+		log.Error().Err(ErrFailedToRegisterListener)
+		return err
+	}
+
+	_, err = e.Emitter.On(OrganizationDeleteOne, handleOrganizationDelete)
+	if err != nil {
+		log.Error().Err(ErrFailedToRegisterListener)
+		return err
+	}
+
+	return nil
+}
+
+// handleOrganizationDelete handles the deletion of an organization and deletes the customer in Stripe
+func handleOrganizationDelete(event soiree.Event) error {
+	client := event.Client().(*entgen.Client)
+	entMgr := client.EntitlementManager
+
+	if entMgr == nil {
+		log.Debug().Msg("EntitlementManager not found on client, skipping customer deletion")
```

Comment:
Use the context logger, e.g. 
```suggestion
		zerolog.Ctx(event.Context()).Msg("EntitlementManager not found on client, skipping customer deletion")
```